blog/disable-alerts-health-management-api.md

```diff
@@ -0,0 +1,147 @@
+---
+slug: Mute Alerts 
+title: "How to mute alerts during maintenance windows or scheduled backups?"
+description: "Disable or Mute Alerts during maintenance windows"
+tags: [netdata,alerts,silence,disable,mute,maintenance window]
+keywords: [netdata,alerts,silence,disable,mute,maintenance window]
+authors: satya
+---
+
+For all SREs, it is absolutely crucial to filter out expected events during maintenance windows and quickly pinpoint critical issues in your infrastructure. Every minute is crucial while dealing with troubleshooting issues and any distractions that may hijack the troubleshooting process should be subdued.
+The health management APIs in Netdata allows teams to eliminate unnecessary alerting during scheduled maintenance, testing, auto scaling events, and instance reboots.
+ 
+## Health Management API
+
+Netdata v1.12 and beyond provides a command API to control health checks and notifications at runtime. The feature is especially useful for maintenance periods, during which you receive meaningless alerts. From Netdata v1.16.0 and beyond, the configuration controlled via the API commands is [persisted across Netdata restarts](https://learn.netdata.cloud/docs/agent/web/api/health#persistence).
```

Comment:
I think, this far ahead of the version, we can say (for the blog at least), just "Netdata provides a command API..."

Same with the next mention. Let's just say it exists without version specificity. 

You okay with that, @cakrit?

Reply:
Yes, that version is ancient.

Reply:
This is fixed.